test/fixtures/transaction_fixtures.js

```diff
@@ -9,14 +9,19 @@ const pactBase = require(path.join(__dirname, '/pact_base'))
 const pactRegister = pactBase()
 
 const buildChargeEventWithDefaults = (opts = {}) => {
-  return {
+  let chargeEvent = {
```

Comment:
`let` could be a `const` here! No problem with updating attributes, the pointer to the original object is what can't be updated